src/main.js

```diff
@@ -216,14 +126,20 @@ module.exports = {
       ignoredRulesWhenFixing = idsToIgnoredRules(ids)
     }))
 
-    // Initialize the worker during an idle time, with a maximum wait of 5 seconds
-    window.requestIdleCallback(initializeWorker, { timeout: 5000 })
+    const initializeESLintWorker = () => {
+      this.worker = new Task(require.resolve('./worker.js'))
+    }
+    // Initialize the worker during an idle time
+    window.requestIdleCallback(initializeESLintWorker)
```

Comment:
If you specify a timeout the idle callback can skip the queue entirely, so we would need to re-architecture how the entire thing works to ensure that the worker has initialized when needed.